clang/lib/Headers/ia32intrin.h

```diff
@@ -411,13 +497,45 @@ __rdpmc(int __A) {
 /// \param __A
 ///    Address of where to store the 32-bit \c IA32_TSC_AUX value.
 /// \returns The 64-bit value of the time stamp counter.
+/// \see _rdtsc
 static __inline__ unsigned long long __DEFAULT_FN_ATTRS
 __rdtscp(unsigned int *__A) {
   return __builtin_ia32_rdtscp(__A);
 }
 
-#define _rdtsc() __rdtsc()
```

Comment:
I think the old code is correct. The `__rdtsc` is a built in, and we have documented in Intrinsic Guide.

Reply:
Thanks! I will revise the description, and make a note to add `__rdtsc` to our guide. (We have `__builtin_readcyclecounter` but not that one.)